i18n/zh-hant.php

```diff
@@ -0,0 +1,347 @@
+                <a role="button" data-toggle="dropdown" class="btn btn-default"><span class="caret" /></a>
+                <ul class="dropdown-menu dropdown-menu-right">
+                  <li><a onclick="$('#licensei').val('創用CC 姓名標示-相同方式分享 4.0 國際版'); $('#iawattr').show();">創用CC 姓名標示-相同方式分享 4.0 國際版</a></li>
+                  <li><a onclick="$('#licensei').val('創用CC 姓名標示 4.0 國際版'); $('#iawattr').show();">創用CC 姓名標示 4.0 國際版</a></li>
+                  <li><a onclick="$('#licensei').val('創用CC CC0 1.0 通用版'); $('#iawattr').hide();">創用CC CC0 1.0 通用版 (貢獻至公共領域)</a></li>
+                </ul>
+                <a role="button" href="//commons.wikimedia.org/wiki/Commons:First_steps/License_selection" target="_blank" class="btn btn-default">
+                  <span class="glyphicon glyphicon-question-sign" />
+                </a>
+              </div>
+            </div></div>
+            <p>我明白當我以上述授權釋出後，我便允許任何人在其他授權以及法律的限制下，對本作品進行商業或其他的再製使用、或根據他們的需求任意修改。</p>
+            <p>我理解這樣的授權並不只限於維基百科以及其他相關站臺之內。</p>
+            <p id="iawattr">我理解我仍有作品的著作權，而根據我所選擇的授權格式分享。其他人對本作品的修改並不是我著作權的範圍。</p>
+            <p>我理解我不能撤回這項協議，而這份授權作品可能會永久保存於維基媒體計畫中，但也可能無法永久保存於維基媒體計畫中。</p>
+            <br />
+            <button type="button" class="btn btn-default btn-block" data-toggle="button" onclick="$('#iag').toggle();">我同意</button>
+            <br /><br />
+          </div>
+          <div class="col-md-4">
+            <div style="display:none;" id="iag">
+              <a role="button" class="btn btn-primary btn-block nt" onclick="s4v()">前進至下一步</a>
+            </div>
+          </div>
+          <br />
+        </div>
+
+      </form> <!-- result -->
+      <div id="result" class="row hof">
+        <br />
+        步驟 5/5 [ <a href="#s1" class="smsc">從頭開始</a> | <a href="//commons.wikimedia.org/wiki/Commons:Help_desk?action=edit&section=new&preloadtitle=help+with+Wikimedia+OTRS+release+generator+step+5" target="_blank">說明</a> | <a href="//commons.wikimedia.org/wiki/User_talk:FDMS4?action=edit&section=new&preloadtitle=Wikimedia+OTRS+release+generator+feedback" target="_blank">回饋</a> ]
+        <br /><br />
+        <?php if (($s1 != "") && ($name != "") && !(($s1 == "2") && (($rep == "") || ($auth == ""))) && ($s2 != "") && !(($s2 == "1") && ($filer == "")) && ($s3 != "") && ($license != "")) {
+          $stats = fopen("stats/" . date('Y') . ".csv", "a");
+          fputcsv($stats, array (date("m-d"), $starttime, date("H:i:s"), $trn), ";");
+          fclose($stats);
+        ?>
+        <div class="col-md-7">
+          <p>如果您有安裝電子郵件用戶端，只要<b>點選右側的按鈕</b>，就會建立一封釋出授權電子郵件。如果沒有安裝 (或點選過後沒有反應)，請手動複製下方綠色方塊中的文字，貼到您的電子郵件中，寄到 <a href="mailto:permissions-zh@wikimedia.org">permissions-zh@wikimedia.org</a>。</p>
+          <p>這封電子郵件應當要從<b>與其釋出作品內容有相關並可以辨識出來的電子郵件位址</b>寄送出去。舉例，如果您所釋出的圖像於某個網站中展示，您的電子郵件位址應當與該網站或列於該網站的聯絡資訊頁面有所關連；如果您釋出的圖像代表某個組織，您的電子郵件位址應當為該組織的官方電子郵件位址。</p>
+          <br />
+          <?php
+            if ($s1 == "1") {
+                $p1s = "$name 是";
```

Comment:
Disputing the space placement here. People using a `zh` form likely give Chinese names, therefore spaces should not be needed.